docs/dev/table/hive/index.md

```diff
@@ -100,147 +102,203 @@ We are using Hive 2.3.4 and 1.2.1 as examples here.
+       // you can use your own hadoop jars. Either way, make sure it's compatible with your Hadoop
+       // cluster and the Hive version you're using.
+       flink-shaded-hadoop-2-uber-2.7.5-{{ site.shaded_version }}.jar
+
+       // Hive dependencies
+       hive-exec-2.0.0.jar
+
+{% endhighlight %}
 </div>
 
+<div data-lang="Hive 2.1.0" markdown="1">
+{% highlight txt %}
+/flink-{{ site.version }}
+   /lib
 
-Similarly, If you are building your own program, you need the above dependencies in your mvn file.
+       // Flink's Hive connector. Contains flink-hadoop-compatibility and flink-orc jars
+       flink-connector-hive{{ site.scala_version_suffix }}-{{ site.version }}.jar
 
-<div class="codetabs" markdown="1">
-<div data-lang="Hive 2.3.4" markdown="1">
-{% highlight xml %}
-<dependency>
-  <groupId>org.apache.flink</groupId>
-  <artifactId>flink-connector-hive{{ site.scala_version_suffix }}</artifactId>
-  <version>{{site.version}}</version>
-  <scope>provided</scope>
-</dependency>
+       // Hadoop dependencies
+       // You can pick a pre-built Hadoop uber jar provided by Flink, alternatively
+       // you can use your own hadoop jars. Either way, make sure it's compatible with your Hadoop
+       // cluster and the Hive version you're using.
+       flink-shaded-hadoop-2-uber-2.7.5-{{ site.shaded_version }}.jar
+
+       // Hive dependencies
+       hive-exec-2.1.0.jar
 
-<!-- Hadoop Dependencies -->
+{% endhighlight %}
+</div>
 
-<dependency>
-  <groupId>org.apache.flink</groupId>
-  <artifactId>flink-hadoop-compatibility{{ site.scala_version_suffix }}</artifactId>
-  <version>{{site.version}}</version>
-  <scope>provided</scope>
-</dependency>
+<div data-lang="Hive 2.2.0" markdown="1">
+{% highlight txt %}
+/flink-{{ site.version }}
+   /lib
 
-<!-- Pick the correct Hadoop dependency for your project.
-Hive 2.3.4 is built with Hadoop 2.7.2. We pick 2.7.5 which flink-shaded-hadoop is pre-built with,
- but users can pick their own hadoop version, as long as it's compatible with Hadoop 2.7.2 -->
+       // Flink's Hive connector. Contains flink-hadoop-compatibility and flink-orc jars
+       flink-connector-hive{{ site.scala_version_suffix }}-{{ site.version }}.jar
 
-<dependency>
-  <groupId>org.apache.flink</groupId>
-  <artifactId>flink-shaded-hadoop-2-uber</artifactId>
-  <version>2.7.5-{{ site.shaded_version }}</version>
-  <scope>provided</scope>
-</dependency>
+       // Hadoop dependencies
+       // You can pick a pre-built Hadoop uber jar provided by Flink, alternatively
+       // you can use your own hadoop jars. Either way, make sure it's compatible with your Hadoop
+       // cluster and the Hive version you're using.
+       flink-shaded-hadoop-2-uber-2.7.5-{{ site.shaded_version }}.jar
+
+       // Hive dependencies
+       hive-exec-2.2.0.jar
+
+       // Orc dependencies -- required by the ORC vectorized optimizations
+       orc-core-1.4.3.jar
+       aircompressor-0.8.jar
```

Comment:
I didn't see this in our pom file?

Reply:
We depend on `orc-core` in our `hive-2.2.0` profile. `aircompressor` is a transitive dependency of `orc-core`

Reply:
can we add this explanation to documentation? like 
```suggestion
       aircompressor-0.8.jar  // transitive dependency of orc-core
```